src/shared/decodeImageFrame.js

```diff
@@ -143,7 +143,8 @@ function decodeImageFrame(
       callbackFn(postProcessDecodedPixels(imageFrame, options, start));
     })
     .catch((err) => {
-      throw err;
+      console.log('The callback is rejected because', err);
+      callbackFn(err, null, err);
```

Comment:
why we are calling the callback on the error, the callback is minMax calcialtion